amundsen_application/api/jira/v0.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from http import HTTPStatus
+from flask import Response, jsonify, make_response, request
+from flask.blueprints import Blueprint
+from flask import current_app as app
+from typing import Any
+
+from amundsen_application.jira.jira_client import JiraClient
+from amundsen_application.api.utils.request_utils import get_query_param
+
+LOGGER = logging.getLogger(__name__)
+
+jira_blueprint = Blueprint('jira', __name__, url_prefix='/api/jira/v0')
+
+
+@jira_blueprint.route('/issues', methods=['GET'])
+def get_jira_issues() -> Response:
+    """
+    Given a table key, returns all JIRA tickets containing that key. Returns an empty array if none exist
+    :return: List of JIRA tickets
+    """
+    try:
+        table_key = get_query_param(request.args, 'key', 'Request requires a key')
```

Comment:
there isn't any specific logic that ties it only to a table - we're tying the jira ticket to the table by the table uri. so it could be extended to anything with a unique name or id. 

i added parameter descriptions, very good point thank you!